lib/main.coffee

```diff
@@ -46,6 +46,16 @@ module.exports =
       lintOnFly: true
       lint: (textEditor) =>
         filePath = textEditor.getPath()
+        # Check if the project directory contains .coafile
+        check = helpers.exec(@executable,
+                             ['--find-config'],
+                             {cwd: path.dirname(filePath)}).then (result) ->
+          atom.notifications.addSuccess('Check completed').catch (error) ->
+            atom.notifications.addInfo \
+              'Missing Dependencies!',
```

Comment:
should we keep it as `coala: Missing ...` ? The package with missing dependencies should be apparent.